graphql/subcourse/fields.ts

```diff
@@ -1,11 +1,14 @@
-import { Course, Lecture, Subcourse, Pupil, Bbb_meeting as BBBMeeting } from '../generated';
-import { Arg, Authorized, FieldResolver, Query, Resolver, Root } from 'type-graphql';
-import { Prisma } from '@prisma/client';
-import { prisma } from '../../common/prisma';
-import { Role } from '../authorizations';
-import { LimitedQuery, LimitEstimated } from '../complexity';
-import { CourseState } from '../../common/entity/Course';
-import { PublicCache } from '../cache';
+import { Course, Lecture, Subcourse, Pupil } from "../generated";
+import { Arg, Authorized, Ctx, FieldResolver, Query, Resolver, Root } from "type-graphql";
+import { Prisma } from "@prisma/client";
+import { prisma } from "../../common/prisma";
+import { Role } from "../authorizations";
+import { LimitedQuery, LimitEstimated } from "../complexity";
+import { CourseState } from "../../common/entity/Course";
+import { PublicCache } from "../cache";
+import { getSessionPupil } from "../authentication";
+import { GraphQLContext } from "../context";
+import { BBBMeeting } from "../../common/entity/BBBMeeting";
```

Comment:
Yeah the import here was changed, and is wrong

Reply:
Ah ok I see ty